src/shared/components/in-app-search/InAppSearchMatches.tsx

```diff
@@ -157,19 +289,22 @@ const InAppSearchMatch = (props: InAppSearchMatchProps) => {
           onOutsideClick={hideTooltip}
           onClose={hideTooltip}
         >
-          <MatchDetails {...props} onClick={onAppClick} />
+          <GeneMatchDetails {...props} onClick={onAppClick} />
         </PointerBox>
       )}
     </>
   );
 };
 
-const MatchDetails = (
+const GeneMatchDetails = (
   props: Pick<InAppSearchMatchProps, 'match' | 'mode' | 'genomeIdForUrl'> & {
     onClick: (appName?: AppNameForViewInApp) => void;
   }
 ) => {
-  const { match, genomeIdForUrl } = props;
+  const { match, genomeIdForUrl } = props as {
+    match: GeneSearchMatch;
+    genomeIdForUrl: string;
+  };
```

Comment:
The type of props should probably be defined when the function is defined. It seems odd to be redefining the props type here.